pommerman/agents/dummy_agent.py

```diff
@@ -0,0 +1,15 @@
+from collections import defaultdict
+import random
+
+import numpy as np
+
+from . import BaseAgent
+from ..envs import utility
+
+
+class DummyAgent(BaseAgent):
+    """This is a baseline agent. After you can beat it, submit your agent to compete."""
```

Comment:
Maybe something like `"""Starts off dead - used to create 1v1 scenarios"""` would be more applicable